Offical jack arevalo

j2356k opened the pull request.

j2356k committed:
milestone 6 local storage

j2356k committed:
4/21/24

j2356k committed:
Merge branch 'master' of https://github.com/Salvarez99/HealthJournalApp into OfficalJackArevalo

j2356k committed:
updated AddJournalEntryForm

j2356k committed:
not working just yet

j2356k committed:
Merge branch 'master' of https://github.com/Salvarez99/HealthJournalApp into OfficalJackArevalo

j2356k committed:
appoint almost functional

j2356k committed:
appointment work and database opens and only close
if the app is closed, functions for reading and
writing data work.

j2356k committed:
all inputforms are finished and function, all that
is left is displaying the data. fetching
data functions are made just has to be
implemented

j2356k committed:
the following has been intergrated to all input
forms current issue is that when open LocalDatabase
is called it will not initalize. after research
looks like this is a common problem online will be
function once this issue is resolved.

j2356k committed:
Merge branch 'master' of https://github.com/Salvarez99/HealthJournalApp into OfficalJackArevalo

Salvarez99 reviewed (changes requested):
PR should be functional before merger

eapresco committed:
Updated gitignore to include firebase config file

j2356k committed:
Merge branch 'master' of https://github.com/Salvarez99/HealthJournalApp into OfficalJackArevalo

j2356k committed:
fixed a minor common error within the table create
ion function

eapresco committed:
Added SQLite support, LocalDatabase.js is the
database management module, any CRUD operations
(Create, Read, Update, Delete) must be done in
there, then all you have to do is import the
functions into whichever file wants to use them.
The initializeDatabase() function is called in the
App.js file and any changes made it are saved to
the device. Right now only the appointment table
exists. When an appointment is created and saved
using the AddAppointmentForm.js it is saved to the
table and it re-renders the JournalScreen using
the navigator. It will display Journal + the ID
fetched from the table. The rest of the logic we
need to complete should be a lot more
straightforward now!

eapresco committed:
Merge branch 'master' of https://github.com/Salvarez99/HealthJournalApp into Ethan

j2356k committed:
Merge branch 'master' of https://github.com/Salvarez99/HealthJournalApp into OfficalJackArevalo

eapresco committed:
Added new tables and CRUD methods for medicine
and journals, mapping to calendar and medicine is
new!

j2356k committed:
Merge branch 'Ethan' of https://github.com/Salvarez99/HealthJournalApp into OfficalJackArevalo

j2356k committed:
updating functions
